frontend/mr-admin-flate/src/components/tiltaksgjennomforinger/TiltaksgjennomforingSchema.ts

```diff
@@ -149,6 +153,21 @@ export const TiltaksgjennomforingSchema = z
         });
       }
     }
+    if (data.visEstimertVentetid && !data.estimertVentetidEnhet) {
```

Comment:
Kanskje litt ryddigere å gjøre om `estimertVenteidVerdi` og `estimertVentetidEnhet` om til et objekt i stedet ( esitmertVentetid: { verdi: ..., enhet: ... } )? Da kan denne valideringen gjøres på det objektet, i stedet for i "superrefine".

Reply:
Det ble litt mindre kode ved å ha det som et objekt 👍